src/promptflow/promptflow/executor/flow_validator.py

```diff
@@ -34,10 +34,11 @@ def _ensure_nodes_order(flow: Flow):
                     continue
                 if i.value not in dependencies:
                     msg = (
-                        f"Node '{n.name}' references a non-existent node '{i.value}' in your flow. "
-                        f"Please review your flow to ensure that the node name is accurately specified."
+                        "Flow is defined incorrectly. Node '{{node_name}}' references a non-existent node "
```

Comment:
Suggestion:
Flow incorrectly defined: Node...

Reply:
right

Reply:
actually, the issue happens at the references btw diff nodes.  I usually classify this at "flow" level error. For me, node error is within a single node, no involvement of other node or flow things.  But I am open to discuss this if not make sense to you.